doc/developer/building-frr-on-ubuntu1804.rst

```diff
@@ -1,33 +1,34 @@
-Ubuntu 18.04LTS
-===============================================
+Ubuntu 18.04 LTS
+================
 
 Install dependencies
--------------------------
+--------------------
+
 Required packages
 ^^^^^^^^^^^^^^^^^
 
 ::
 
-    sudo apt-get install \
-        git \
-        autoconf \
-        automake \
-        libtool \
-        make \
-        gawk \
-        libreadline-dev \
-        texinfo \
-        pkg-config \
-        libpam0g-dev \
-        libjson-c-dev \
-        bison \
-        flex \
-        python-pytest \
-        libc-ares-dev \
-        python3-dev \
-        libsystemd-dev \
-        python-ipaddr \
-        python3-sphinx
+   sudo apt-get install \
+       git \
+       autoconf \
+       automake \
+       libtool \
+       make \
+       gawk \
+       libreadline-dev \
+       texinfo \
+       pkg-config \
+       libpam0g-dev \
+       libjson-c-dev \
+       bison \
+       flex \
+       python-pytest \
+       libc-ares-dev \
+       python3-dev \
+       libsystemd-dev \
+       python-ipaddr \
+       python3-sphinx
```

Comment:
Note: In my Docker container for topotest I also need `install-info` to be able to compile FRR.

Reply:
Yeah actually this list is all kinds of screwed up because some of these are build deps and some of them are regular deps, and if we're listing build deps we certainly don't list all of them (where is `build-essential`?). But I am 99% sure this is the place to list build dependencies so I am going to add `install-info` here pending a full review of all of these.